psh/mem/mem.c

```diff
@@ -300,7 +300,7 @@ static int psh_sharedMaps(void)
 			continue;
 		}
 
-		printf("\nMap #%d\n", info.maps.map[i].id);
+		printf("\nMap #%d: %s\n", info.maps.map[i].id, info.maps.map[i].name);
```

Comment:
Should a length specifier also be used, as the field is of fixed size? Or maybe we should just abide by the syscall's contract (the string is always null-terminated)?

Reply:
The syscall implementation always null-terminates the string, I'm not sure if it's necessary.